src/srf/ratpoly.cpp

```diff
@@ -13,84 +13,31 @@
 // and convergence should be fast by now.
 #define RATPOLY_EPS (LENGTH_EPS/(1e2))
 
-double SolveSpace::Bernstein(int k, int deg, double t)
+// indexed by [degree][k][exponent]
+const double bernstein_coeff[4][4][4] = {
```

Comment:
These should probably be `static` too, for the same reason the functions are.

Reply:
Updated.